JapaneseFilter/sections/specific.txt

```diff
@@ -45,6 +45,7 @@ hentai-covers.site,109cinemas.net,4545.to,bunshun.jp,eikaiwa.dmm.com,gizmodo.jp,
 !
 seleck.cc##div[class$="sticky-sidebar__widgets"]
 city.tochigi.lg.jp###top_banner_footer
+agora-web.jp##iframe[style="width:300px;height:250px;border:none;vertical-align:bottom;"]
```

Comment:
iframe is inserted by JS. Better block by a scriptlen in addition.

Reply:
Ok. What I will add in addition
```
! https://github.com/AdguardTeam/AdguardFilters/pull/136936
agora-web.jp#%#//scriptlet('abort-current-inline-script', 'document.currentScript', 'insertAdjacentHTML')
```

Reply:
thank you so much.
it was a good experience.